### DISCUSSION TITLE
legendary --language xx-XX?

### DISCUSSION THREAD

**Discussion by DocMAX** [legendary-gl/legendary#642] [General]
Shouldn't this change the language in games automatically? Not the case for me. I use --language de-DE. Is there something wrong?

**Comment by arielj** [comment]
not all games support that flag

for example, Rocket League uses a different pattern, you need to provide a game argument with a single `-` and a 3-letters language code https://www.epicgames.com/help/en-US/c-Category_RocketLeague/c-RocketLeague_Gameplay/how-do-i-change-my-language-in-rocket-league-a000084576

for some games you have to provide a 2-letters language code, for some it doesn't work at all if the game expects you to change that from their settings

it's up to each game developer to support that feature or not

**Comment by DocMAX** [comment]
How can i find out the right flag for each of my games?

**Reply by arielj** [reply]
> How can i find out the right flag for each of my games?

I don't think there's a place with this information for all games, like... that link for rocket league is in the epic's help section

you'll have to google things for each of the games you want
